src/lib/drivers/tlog/writer.rs

```diff
@@ -191,8 +191,46 @@ impl DriverInfo for TlogWriterInfo {
     }
 
     fn create_endpoint_from_url(&self, url: &url::Url) -> Option<Arc<dyn Driver>> {
+        let expected_system_id = url.query_pairs().find_map(|(key, value)| {
+            if key != "system_id" {
+                return None;
+            }
+            value.parse::<u8>().ok()
+        });
+
+        const MAV_COMP_ID_AUTOPILOT1: u8 = 1; // From: https://mavlink.io/en/messages/minimal.html#MAV_COMPONENT
```

Comment:
you can get it from: `mavlink::ardupilotmega::MavComponent::MAV_COMP_ID_AUTOPILOT1`

Reply:
ty!